.github/workflows/benchmark.yml

```
@@ -0,0 +1,43 @@
+name: Benchmarks
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron:  "0 7 * * 1,3"
```

In description, you wrote ` fired every monday,`
```suggestion
    - cron:  "0 7 * * 1"
```